cni/network/network.go

```diff
@@ -487,26 +418,36 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 		}
 	}
 
+	switch nwCfg.Ipam.Type {
```

Comment:
This is coming from the json that gets laid down with the cni plugin I assume? 

Are we allowed to look for the extiance of any other files to make this decision?